templates/default/cgroup.conf.erb

```diff
@@ -0,0 +1,7 @@
+###
+# Slurm cgroup support configuration file
+###
+CgroupAutomount=yes
+<% if node['conditions']['old_cgroup_directory'] -%>
```

Comment:
use https://docs.chef.io/resources/template/#template_specificity instead of condition based on platform